package.json

```diff
@@ -579,6 +579,10 @@
                     "command": "cosmosDB.openInPortal",
```

Comment:
You still have `.DS_Store` files in this PR. Please remove them - you might need to google how to do it if git is giving you problems (sometimes happens with files that you added before you changed your gitignore)